string[] names
float32List[] stiffnessLists
float32List[] timeLists
---
bool res